### PR TITLE
feat: add a heatmap

### PR DESCRIPTION
**Pull request by JFrankfurt**
|desktop|mobile|
|-|-|
|image|image|

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

**Review by ricardoMogg — APPROVED** [review]

**Comment by 0xchiaroscuro** [comment]
color for gray squares should be #F8F9FB. 

Also can we apply less spacing between the 2 rows of data? looks really spacious rn. Attaching designs 

Screenshot 2024-10-01 at 4 48 49 PM
Screenshot 2024-10-01 at 4 48 58 PM

**Comment by 0xchiaroscuro** [comment]
attaching color scale

File: apps/web/.env.local.example [hunk 1]
```diff
@@ -23,4 +23,8 @@ DATADOG_CLIENT_ID=
 NEYNAR_API_KEY=
 
 FARCASTER_DEVELOPER_FID=
-FARCASTER_DEVELOPER_MNEMONIC=
\ No newline at end of file
+FARCASTER_DEVELOPER_MNEMONIC=
+
+ETHERSCAN_API_KEY=
+BASESCAN_API_KEY=
+TALENT_PROTOCOL_API_KEY=
```
Comment (omkarb) [inline apps/web/.env.local.example:30]: Do these keys need to be added to config service or are they already in there?

**Review by omkarb — APPROVED** [review]

**Comment by 0xchiaroscuro** [comment]
Screenshot 2024-10-01 at 6 15 27 PM (1)

File: apps/web/.env.local.example (hunk 1 above)
Comment (JFrankfurt) [reply apps/web/.env.local.example:30]: these still need to be added via codeflow 👍

**Review by 0xchiaroscuro — APPROVED** [review]

**Comment by cb-heimdall** [comment]
This Pull Request was merged without enough reviews
Please go [here](https://github.com/base-org/web/issues/1031) to review and +1

**Comment by cb-heimdall** [comment]
This Pull Request was merged without enough reviews
Please go [here](https://github.com/base-org/web/issues/1031) to review and +1

**Pull request merged**